tests/integration/test_models.py

```diff
@@ -217,3 +217,107 @@ class TestScikitRegression(IntegrationCLITestCase):
         self.assertIn("value", results)
         results = results["value"]
         self.assertTrue(results is not None)
+
+
+class TestScikitClustering(IntegrationCLITestCase):
+    async def test_run(self):
+        self.required_plugins("dffml-model-scikit")
+        # Create the training data
+        train_filename = self.mktempfile() + ".csv"
+        pathlib.Path(train_filename).write_text(
+            inspect.cleandoc(
+                """
+                crim,zn,indus,chas,nox,rm,age,dis,rad,tax,ptratio,b,lstat
```

Comment:
Let's generate a random dataset to use for this. It looks like this might be an existing dataset.

Reply:
Done.